Improve Docker Compose README.md (#10444)
apache/pulsar: docker-compose/standalone-dashboard/README.MD

-->
# Docker Compose

> To start the cluster, run the following command in this folder: 

Reviewer: in which folder?

Author: Replaced "this" with "the current".

```
docker-compose up
```

## Available endpoints

> The following endpoints will be available (assuming Docker Compose exposes ports on `localhost`):

Reviewer: ```suggestion
The following endpoints are available (assuming Docker Compose exposes ports on `localhost`):
```

Author: Applied

* [web-dashboard](http://localhost:80) - Pulsar dashboard, shows various metrics and metadata information about the cluster
* `http://localhost:8080` - broker HTTP REST API URL, for both administrative tasks and for topic lookup by producers and consumers

> * `pulsar://locahost:6650` - broker binary protocol URL, for all data transfers by producers and consumers

Reviewer: ```suggestion
* `pulsar://locahost:6650` - broker binary protocol URL, for all data transferred by producers and consumers
```

Author: Applied
